subsys/bluetooth/host/Kconfig

```diff
@@ -509,6 +509,14 @@ config BT_KEYS_SAVE_AGING_COUNTER_ON_PAIRING
 	  time a successful pairing occurs. This increases flash wear out but offers
 	  a more correct finding of the oldest unused pairing info.
 
+config BT_SMP_MIN_ENC_KEY_SIZE
+	int "Minimum encryption key size accepted in octets"
+	range 7 16
+	default 7
```

Comment:
Do we want to lock this to 16 when `BT_SMP_SC_ONLY` is enabled?

Reply:
That makes sense, there would be other places where we check the encryption key size that could be removed in that case.

Reply:
Isn't it possible to have a LE SC key that's less than 128-bit?

Reply:
Secure Connections Only Mode (`BT_SMP_SC_ONLY`) is a different concept than LE Secure Connections.

Reply:
Ah, I had that confused with `BT_SMP_SC_PAIR_ONLY` :)